### PR TITLE
Update proposals that are implemented in Swift 6.0.

### PR DESCRIPTION
**Pull request by hborla**
Double checking that I'm not missing others...

### CODE REVIEW

File: proposals/0429-partial-consumption.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 * Authors: [Michael Gottesman](https://github.com/gottesmm), [Nate Chandler](https://github.com/nate-chandler)
 * Review Manager: [Xiaodi Wu](https://github.com/xwu)
 * Implementation: On `main` gated behind `-enable-experimental-feature MoveOnlyPartialConsumption`
```
Comment (nate-chandler) [inline proposals/0429-partial-consumption.md:6]: Should this line be deleted or replaced with a link to https://github.com/swiftlang/swift/pull/73159 ?

Comment (hborla) [reply proposals/0429-partial-consumption.md:6]: Ah, yes, we can just remove the implementation field

File: proposals/0425-int128.md [hunk 2]
```diff
@@ -3,8 +3,7 @@
 * Proposal: [SE-0425](0425-int128.md)
 * Author: [Stephen Canon](https://github.com/stephentyrone)
 * Review Manager: [Doug Gregor](https://github.com/DougGregor)
-* Implementation: https://github.com/stephentyrone/swift-numerics/tree/int128
```
Comment (benrimmington) [inline proposals/0425-int128.md:6]: The "int128" branch details could also be removed from the §"Proposed solution".

**Review by stephentyrone — APPROVED** [review]

**Review by Azoy — APPROVED** [review]
Thank you!

**Pull request merged**